### PR TITLE
SONARPY-2068: Tabulations (\t) should be handled correctly as an escape character

### PR DESCRIPTION
**Pull request by Seppli11**

### CODE REVIEW

File: sonar-python-plugin/src/test/java/org/sonar/plugins/python/TestUtils.java [hunk 1]
```diff
@@ -59,4 +62,11 @@ public static PythonInputFile createInputFile(File baseDir, String name, InputFi
       .build());
   }
 
+  public static List<ColumnMapping> mapToColumnMappingList(Map<Integer, Integer> map) {
```
Comment (Seppli11) [inline sonar-python-plugin/src/test/java/org/sonar/plugins/python/TestUtils.java:65]: Do you have a `TestUtils` that is accessible in all tests?

File: sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonHighlighterTest.java [hunk 2]
```diff
@@ -211,14 +212,20 @@ void number() {
 
   @Test
   void highlightingNotebooks() {
-    String pythonContent = "def foo():\n    pass\na = \"test\" # comment\n# test \\n \\\\n test\nb = 3J\n#SONAR_PYTHON_NOTEBOOK_CELL_DELIMITER";
+    String pythonContent = """
```
Comment (joke1196) [inline sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonHighlighterTest.java:215]: Maybe in the highlighter we could add a `\t` to one of the test. To be sure the impact of the changes you made are propagated here.

**Review by joke1196 — CHANGES_REQUESTED** [review]
Looks really good. Thanks for the clean up! I have added a small suggestion for the naming as well as small change on test. As stated in my comment we should add code for something that we do not support (unicode escape sequences), but I think this is fine like so as it is a way better version than what was there already :) (which already had code for something we were not using)

**Review by joke1196 — APPROVED** [review]
LGTM! Thanks a lot for this. This clean up was really needed!

**Pull request merged**